src/pageScanner/checks/transcript-missing.js

```
@@ -0,0 +1,87 @@
+	if ( figure ) {
+		const figcaption = figure.querySelector('figcaption');
+		if ( figcaption ) {
+			text += figcaption.textContent.trim() + ' ';
+		}
+	}
+
+	// Try to find more specific containing element first
+	const parent = node.closest( 'figure, .media-wrapper, section, article, div, body' );
+	if ( parent ) {
+		// Skip hidden nodes
+		const nodeFilter = function(node) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)